compiler/next/include/chpl/uast/ASTClassesList.h

```diff
@@ -130,7 +131,7 @@ AST_BEGIN_SUBCLASSES(Sym)              // old AST: Symbol
 
   AST_BEGIN_SUBCLASSES(TypeSym)        // old AST: TypeSymbol/Type
     AST_NODE(Enum)                     // old AST: EnumType
-    AST_NODE(SimpleType)               //
+    //AST_NODE(SimpleType)             //
```

Comment:
If commenting out `SimpleType` is intended, please explain in the PR message, esp. why still keep this line in the source code.